api.md

```diff
@@ -878,6 +884,131 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
 
 runs a reorder on the `_timescaledb_internal._hyper_1_10_chunk` chunk using the `conditions_device_id_time_idx` index.
 
+---
+## Compression :community_function: [](Compression)
+TimescaleDB allows users to configure thier hypertable for compression and compress chunks of data manually or based 
+on policy.  Policy based compression will allow the user to compress a chunk when it has reached the specified age.
+The user may also manually compress a chunk or sets of chunks.  For more detailed discussion of this fuction <insert 
+tag to tutorial>.
+
+>:WARNING: Compression is disabled when running postgres 9.6 you must upgrade your Postgresal instance to Postgresql 10
+ or later in order to use this feature. 
+ 
+ #### Restrictions
+ Version 1.5 does not support altering or inserting data into compressed chunks.  The data can be queried without any 
+ modifications, however if you need to backfill or update data in a compressed chunk you will need to decompress the chunk(s)
+ first.
+
+
+*	[Compress Table ORDER BY] (#Compress_table-orderby)
+*	[Compress Table SEGMENT BY] (#Compress_table-segmentby)
+*	[Compress Chunk](#Compress_chunk)
+*	[Compress Policy] (#Compress_policy)
+*	[Compress Policy Remove] (#Compress_policy_remove)
+*	[Decompress Chunk] (#Decompress_Chunk)
+
+## Compress Table ORDER BY (Compression) :community_function: [](#Compress_table-orderby)
```

Comment:
I wonder if this should really be a `ALTER TABLE (Compression)` section modeled after what we already have for "ALTER VIEW (Continuous Aggregate) " That way order by and segment_by become options within the command. This may be more intuitive and follows established conventions.